common-files/test/utils.test.mjs

```diff
@@ -119,8 +120,24 @@ describe('Utils tests', function () {
     assert.equal(result, '');
   });
 
-  it("Shouldn't obfuscate for the Dev environment", () => {
-    process.env.NODE_ENV = 'developer';
+  it("Shouldn't obfuscate for the local environment", () => {
+    process.env.ENVIRONMENT = 'local';
```

Comment:
I don't think there's an environment called `local`, they should match the ones you find in `config.ENVIRONMENTS`. If needed, add one, but we should be consistent

Reply:
Hey @signorecello, the thing with this is that for being considered local, it should be different from 'aws', then I could use whatever value since it isn't equal to 'aws'.